test/signs/realtime_test.exs

```diff
@@ -162,6 +162,10 @@ defmodule Signs.RealtimeTest do
         datetime(~T[05:00:00])
       end)
 
+      stub(Engine.ScheduledHeadways.Mock, :get_last_scheduled_departure, fn _ ->
+        datetime(~D[2023-01-02], ~T[02:00:00])
```

Comment:
Just to be clear - the first scheduled departure should _never_ be after the last scheduled departure right? The API will never return that state?